docs/concepts/advanced/html_fragments.md

```diff
@@ -136,7 +136,7 @@ class MyPage(Component):
 
     class View:
         def get(self, request):
-            return self.component.render_to_response(request=request)
+            return self.component_cls.render_to_response(request=request)
```

Comment:
As I was going through all places where we instantiate Components, I realized that we also do so when executing view functions like `Component.View.get()`.

There, we make a dummy instance and assign it to `View.component`.

But with this PR, it no longer makes sense to manually instantiate Components.

It's also a bit misleading, because `View.get()` is called BEFORE any component is rendered.

So instead of having the access to the dummy component instance, I added this `View.component_cls` field that points just to component CLASS, which better reflects what's actually avilable.